tools/execsnoop.py

```diff
@@ -29,13 +29,14 @@
 
 # arguments
 examples = """examples:
-    ./execsnoop           # trace all exec() syscalls
-    ./execsnoop -x        # include failed exec()s
-    ./execsnoop -T        # include time (HH:MM:SS)
-    ./execsnoop -t        # include timestamps
-    ./execsnoop -q        # add "quotemarks" around arguments
-    ./execsnoop -n main   # only print command lines containing "main"
-    ./execsnoop -l tpkg   # only print command where arguments contains "tpkg"
+    ./execsnoop               # trace all exec() syscalls
+    ./execsnoop -x            # include failed exec()s
+    ./opensnoop -c ./mappath  # only trace cgroups in this BPF map
```

Comment:
In trace.py, we already have `-c` to represent a cgroup path such that any processes in the downward hierarchy of that cgroup path will be traced. Here `-c` means different things. I would like `-c` to be consistent across all tools.

Also, in bpf, we have cgroup_array_map which actually helps reference counted cgroup so cgroup won't go away during tracing. The cgroup map here is just a hash or array map, but user could also pin a cgroup array map (BPF_CGROUP_ARRAY) and use it in the bpf program. The advantage of this is cgroup won't go away doing tracing. The drawback is it is not able to trace arbitrary cgroups. So we probably needs both in the future.

So I recommend we just use long name `--cgroupmap` for this option.  In the future, we can implement `-c` for BPF_CGROUP_ARRAY style implementation (similar to trace.py) or `--cgrouparray` for pinned cgroup array.

Reply:
It makes sense, I'll keep the long name `--cgroupmap` and remove the short name.

If this PR gets accepted, I would like to follow-up with the same cgroup filtering in some other tools (capable, tcpconnect, tcpaccept...). And `tcpconnect` already has a `-c` option too. So that's another argument for using the long name only.

Reply:
sounds good then.